enclave/src/lib.rs

```diff
@@ -425,6 +425,79 @@ pub unsafe extern "C" fn accept_pdex_accounts(
     sgx_status_t::SGX_SUCCESS
 }
 
+extern "C" {
+    fn ocall_send_nonce(
+        ret_val: *mut sgx_status_t,
+        account_encoded: *const u8,
+        account_size: u32,
+        nonce: u32,
+    ) -> sgx_status_t;
+
+    pub fn ocall_send_balances(
+        ret_val: *mut sgx_status_t,
+        account_encoded: *const u8,
+        account_size: u32,
+        token_encoded: *const u8,
+        token_size: u32,
+        free: *mut u8,
+        reserved: *mut u8,
+        balance_size: u32,
+    ) -> sgx_status_t;
```

Comment:
I think the orderbook mirror should also be included here.

Reply:
included